src/node_options.cc

```diff
@@ -466,10 +466,7 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             kAllowedInEnvvar);
   AddAlias("--loader", "--experimental-loader");
   AddOption("--experimental-modules", "", NoOp{}, kAllowedInEnvvar);
-  AddOption("--experimental-wasm-modules",
-            "experimental ES Module support for webassembly modules",
-            &EnvironmentOptions::experimental_wasm_modules,
-            kAllowedInEnvvar);
+  AddOption("--experimental-wasm-modules", "", NoOp{}, kAllowedInEnvvar);
```

Comment:
I think "unflag an experimental feature" should start with switching the default value to be `true`, and allowing turning the feature off with `--no-experiemental-wasm-modules`, instead of making it a no-op directly.

Reply:
I agree, if having WASM module enabled breaks something, it's useful to be able to disable it. We can make the falg a no-op in another major

Reply:
I don't think there is any chance of a break here, as it's a feature that only applies if you explicitly use it, instead I've removed the `EnvironmentOptions::experimental_wasm_modules` and thanks for pointing that out.

Final reviews very welcome, I would like to land this one soon.